packages/loot-core/src/server/sync/encoder.ts

```diff
@@ -2,6 +2,7 @@ import * as encryption from '../encryption';
 import { SyncError } from '../errors';
 import * as prefs from '../prefs';
 
+// @ts-expect-error sync_pb is not a module
```

Comment:
`sync_pb` is too much of a beast to covert it to ESM...

Reply:
Can't you just regenerate `sync_pb` by recompiling `sync.proto`?